go.mod

```diff
@@ -4,6 +4,8 @@ go 1.13
 
 require (
 	github.com/ONSdigital/dp-api-clients-go v1.4.0
+	github.com/ONSdigital/dp-cookies v0.0.0-20200218165833-32f13bf75fbb
```

Comment:
After yesterdays meeting - should we now be release tagging this?

Reply:
My suggestion would be tagging dp-cookies with a 0.1.0 pre-release tag.